### PR TITLE
Newsletter 13: Veloren

### PR DESCRIPTION
**Pull request by AngelOnFira**
This is for #232 :ok_hand:

### CODE REVIEW

File: content/posts/newsletter-013/index.md [hunk 1]
```diff
@@ -124,6 +124,62 @@ Chillscapes is written using an early-in-development 2d engine,
 [neoc]: https://itch.io/jam/neoc03-rhythm-jam
 [kludgine]: https://github.com/khonsulabs/kludgine
 
+### [Veloren][veloren]
+
+![Landscape](veloren-landscape1.png)
+
+^ _Landscape with new LoD and lighting_
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-013/index.md:131]: Some CSS has been added in #264 to make captions format nicely, so you don't need the arrow any more - just make sure to not leave a gap between the image and the caption, or markdown will split them into seperate paragraphs

```suggestion
_Landscape with new LoD and lighting_
```

File: content/posts/newsletter-013/index.md [hunk 2]
```diff
@@ -124,6 +124,62 @@ Chillscapes is written using an early-in-development 2d engine,
...
+The largest merge in Veloren so far also happened in August. It included
+monumental changes to lighting and added level of detail functionality to see
+far-off mountains. Lots of work has been done on the animation, combat, SFX, and
+UX front. Animations for movement and combat were added and improved. Work
+continued on particle systems, which have been added to Veloren in places like
+campfires, fireworks, and weapons.
+
+![Healing sceptre](veloren-sceptre.gif)
+
+^ _Healing sceptre with the new particle system_
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-013/index.md:151]:
```suggestion
_Healing sceptre with the new particle system_
```

File: content/posts/newsletter-013/index.md [hunk 3]
```diff
@@ -124,6 +124,62 @@ Chillscapes is written using an early-in-development 2d engine,
...
+
+In September, work on 0.8 will continue. Some large systems being worked on
+include networking, improved persistence stability, and player experience. Game
+design is working on improving the connection between the experience a new
+player has, and the current game design. The in-progress 0.8 version will likely
+be completed more quickly than 0.7, as to not include too many changes.
+
+![Caves](veloren-cave.png)
+
+^ _Improvements to cave generation_
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-013/index.md:178]:
```suggestion
_Improvements to cave generation_
```

**Review by 17cupsofcoffee — APPROVED** [review]
Content looks good, just a few tweaks to the image captions to take advantage of #264.

**Comment by ozkriff** [comment]
@AngelOnFira don't you mind if we remove the last image to scale down the section a little?

**Comment by AngelOnFira** [comment]
@ozkriff no worries, whatever works well for you :ok_hand:

**Review by ozkriff — APPROVED** [review]
:+1: (made a few style changes and removed the last image)

**Comment by ozkriff** [comment]
Merged, thanks for yet another great PR! :)

**Pull request merged**